test/MusicStore.Test/ShoppingCartControllerTest.cs

```diff
@@ -27,6 +30,8 @@ public ShoppingCartControllerTest()
                       .AddInMemoryStore()
                       .AddDbContext<MusicStoreContext>();
 
+            services.AddMvc();
```

Comment:
Oh, yes, the form token. But then if it's passed in as part of the form body (is it?), won't the regular validation attribute accept it? Or is it getting sent in a query string?